starboard/raspi/2/skia/toolchain/BUILD.gn

```diff
@@ -17,20 +17,6 @@ import("//build/toolchain/gcc_toolchain.gni")
 import("//starboard/build/toolchain/cobalt_toolchains.gni")
 import("//starboard/raspi/shared/toolchain/raspi_shared_toolchain.gni")
 
-gcc_toolchain("target") {
```

Comment:
It's still needed to build Starboard layer for Evergreen ( arm-hardfp ) so this doesn't seem like submittable.

Reply:
yup, i realized it a while back when i ran tests. will kill it